modules/oe_translation_epoetry/tests/src/FunctionalJavascript/EpoetryTranslationTest.php

```diff
@@ -2547,6 +2547,51 @@ public function testAutomatedRequestStatusUpdate(): void {
     }
   }
 
+  /**
+   * Tests that warning message is shown for requests with a weekend deadline.
+   */
+  public function testWeekendDeadlineWarning(): void {
+    $node = $this->createBasicTestNode();
+    $this->drupalGet($node->toUrl('drupal:content-translation-overview'));
+    $this->clickLink('Remote translations');
+    // Select a language.
+    $this->getSession()->getPage()->checkField('Bulgarian');
+    // Select a weekend deadline.
+    $this->getSession()->getPage()->fillField('translator_configuration[epoetry][deadline][0][value][date]', '08/26/2023');
```

Comment:
@22Alexandra This is not a good date to use. In a few months, this test will fail because the date will be in the past